content/cli/users.md

```diff
@@ -231,7 +227,7 @@ Requested user info:
 #### Usage 
 `cfy users deactivate [OPTIONS] USERNAME`
 
-Deactivate a user. Suspends a user's access, without deleting their details.
+Deactivate a user. The user will not be able to log into Cloudify, but will not be deleted altogether. You can later on reactivate the user. 
```

Comment:
Deactivated users cannot login to Cloudify, but are in the list of users. To let the user login to Cloudify, reactivate the user.